test/controllers/users/omniauth_callbacks_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class Users::OmniauthCallbacksControllerTest < ActionController::TestCase
+
+  test "github signup" do
+    OmniAuth.config.add_mock(:github, {:uid => '12345'})
```

Comment:
I think we can use `minitest-rspec_mocks` if we want mocking & stubbing. IMO, `rspec-mocks` provides great mocking features when we compare with `minitest`. 

Thoughts please?